python/feathub/processors/flink/table_builder/datagen_utils.py

```diff
@@ -99,6 +100,6 @@ def get_table_from_data_gen_source(
         else:
             raise FeathubException(f"Unknown field config type {type(field_config)}.")
 
-    t_env.create_temporary_table(data_gen_source.name, table_descriptor_builder.build())
-
-    return t_env.from_path(data_gen_source.name)
+    table_name = generate_random_table_name(data_gen_source.name)
```

Comment:
Would it be useful to update `generate_random_table_name()` so that it is easy for us to programmably derive the original table name from the random table name?
 
For example, it can be in the format {original_table_name}-{uuid}.